schemas/device/MRIScannerUsageForDiffusion.schema.tpl.json

```diff
@@ -1,22 +1,22 @@
 {
-    "_extends": "/neuroimaging/MRIScannerUsage.schema.tpl.json",
-    "_type": "https://openminds.ebrains.eu/neuroimaging/MRIScannerUsageForDiffusion",
-    "properties": {
-        "bValues": {
-            "_instruction": "Add the b values coresponding to this aquisition.",
-            "_linkedTypes": [
-                "https://openminds.ebrains.eu/core/File"
-            ]
-        },
-        "bVectors": {
-            "_instruction": "Add the b vectors coresponding to this aquisition.",
-            "_linkedTypes": [
-                "https://openminds.ebrains.eu/core/File"
-            ]
-        }
+  "_extends": "/neuroimaging/MRIScannerUsage.schema.tpl.json",
+  "_type": "https://openminds.ebrains.eu/neuroimaging/MRIScannerUsageForDiffusion",
```

Comment:
why is this distinction needed? let's discuss if one schema is enough with optional properties for diffusion

Reply:
That was @apdavison 's initial solution too, but I think `bValues` and `bVectors` are really important and should be required somehow.
Anyway, if you both think having it like this is weird, we can revert to having one `MRIScannerUsage` and make the `bValues` and `bVectors` optional.

Reply:
Moved to MRI scanner usage